pkgs/development/libraries/lpcnetfreedv/default.nix

```diff
@@ -1,30 +1,19 @@
 { lib, stdenv, fetchFromGitHub, fetchurl, cmake, codec2 }:
 
-let
```

Comment:
Without this, CMake attempts to fetch the data during the build on my machine (NixOS on x86-64) and fails.

Reply:
Reverted. Thanks!